int32 status

int32 position

int32 speed
int32 torque
int32 temp
int32 voltage